### DISCUSSION TITLE
How to update the window only when input received

### DISCUSSION THREAD

**Discussion by ThomasAlban** [bevyengine/bevy#8719] [Q&A] [answered]
Is it possible to make Bevy only update the window when inputs such as moving the mouse or clicking are received? I'm making a 3d editor using Bevy and it is currently consuming lots of CPU even when running in the background. If not, what other steps can I take to reduce CPU consumption?

**Comment by alice-i-cecile** [comment, marked as answer]
You want [`UpdateMode::ReactiveLowPower`](https://docs.rs/bevy/latest/bevy/winit/enum.UpdateMode.html#variant.ReactiveLowPower). This was added by the Foresight team, who are making a CAD application in Bevy so their use case is very similar.

I also recommend pairing this with [`bevy_framepace`](https://github.com/aevyrie/bevy_framepace) to smooth out input latency :)

**Reply by ThomasAlban** [reply]
Thanks for this, this is exactly what I need. Do you recommend using `ReactiveLowPower` when the window is unfocused and using `Continuous` when it is focused? I tried using `ReactiveLowPower` all the time but since I'm using a WASD controlled camera, holding a key doesn't count as an update so the window would not update while moving.

**Reply by alice-i-cecile** [reply]
Hmm, that's a good question. I would start with that.
